tests/sagemaker/test_sagemaker_plugin.py

```diff
@@ -0,0 +1,615 @@
+import os
```

Comment:
Most of the content in this file are copied from https://github.com/mlflow/mlflow/blob/master/tests/sagemaker/test_deployment.py.

The only changes are:
- Call `sagemaker_plugin.create_deployment()` instead of `mfs.deploy()`
- Create a fixture for the SageMakerPlugin